ui-sdk-components-sample/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.karhoo.samples.networksdk">
```

Comment:
The package can be updated to use uisdk.components

Reply:
Done